src/vws/exceptions.py

```diff
@@ -5,11 +5,33 @@
 import json
 from urllib.parse import urlparse
 
-import requests
 from requests import Response
 
 
-class UnknownVWSErrorPossiblyBadName(Exception):
+class BaseVWSException(Exception):
+    """Base class for Vuforia exceptions."""
+
+
+class VWSException(BaseVWSException):
+    """Exception that has the Vuforia response as a property."""
```

Comment:
In this codebase we use:

```python
"""
...
"""
```

rather than:

```python
"""..."""
```

even for single line docstrings.